x/ccv/types/genesis.go

```diff
@@ -8,11 +8,11 @@ import (
 	abci "github.com/cometbft/cometbft/abci/types"
 )
 
-// NewInitialGenesisState returns a consumer GenesisState for a completely new consumer chain.
+// NewInitialGenesisState returns a ConsumerGenesisState for a completely new consumer chain.
```

Comment:
Maybe also rename this for consistency to NewInitialConsumerGenesisState?

Reply:
ok